### PR TITLE
WASM-ify hello-jniCallback

### PR DESCRIPTION
**Pull request by matthias-blume**

### CODE REVIEW

**Comment by matthias-blume** [comment]
Because of the JNIEXPORT thing this also doesn't quite work yet.
(To make __attribute__((used)) work we seem to need to pass --export-dynamic to the linker.  But when I do that it also tries to export a global variable - namely g_ctx, and there is a bug in our wasm2c, so this case is not handled correctly.)

File: hello-jniCallback/README.md [hunk 1]
```diff
@@ -17,14 +17,32 @@ with C++ support.
...
-1. Open the sample directory.
-1. Open *File/Project Structure...*
+2. Set up the [WASM-NDK toolchain](https://github.com/AndroidWasm/wasm_ndk)
+3. Manually compile the C++ code to WASM:
+
+   ```
+   cd app/src/main/cpp
+   mkdir build_wasm
+   cd build_wasm
+   $WASM_NDK/cmake/toolchain/android_wasm.toolchain.cmake ..
```
Comment (dimitry-) [inline hello-jniCallback/README.md]: cmake --toolchain=$WASM_NDK...

**Review by dimitry- — APPROVED** [review]

File: hello-jniCallback/README.md (hunk 1 above)
Comment (matthias-blume) [reply hello-jniCallback/README.md]: Ugh, yeah - cut&paste error.

Comment (matthias-blume) [reply hello-jniCallback/README.md]: done

**Pull request merged**